index.bs

```diff
@@ -418,20 +426,28 @@ partial interface ModelBuilder {
             - *autoPad*: an {{AutoPad}}. The automatic input padding options. By default, this argument is set to *"explicit"*, which means that the values in the *options.padding* array should be used for input padding. When the option is set other than *"explicit"*, the values in the *options.padding* array are ignored. With the *"same-upper"* option, the padding values are automatically computed such that the additional ending padding of the spatial input dimensions would allow all of the input values in the corresponding dimension to be filtered. The *"same-lower"* option is similar but padding is applied to the beginning padding of the spatial input dimensions instead of the ending one.
             - *transpose*: a {{boolean}} indicating that a transposed convolution operation is performed. Transposed convolution is used in upsampling networks to increase the resolution of a feature as opposed to the typical convolution process that reduces the feature's resolution. When transposed convolution is performed, *options.outputPadding* may be needed to disambiguate the output tensor shape. If not present, this option is assumed to be false.
             - *groups*: a {{long}} scalar. The number of groups that input channels and output channels are divided into, default to 1.
-            - *layout*: an {{OperandLayout}} with value as *"nchw"* or *"nhwc"*. The default value is *"nchw"*. This argument specifies the layout format of the input, output and filter tensor. Specifically,
+            - *bias*: an {{Operand}}. The 1-D tensor of the size equal to the number of output channels, one value for each channel. A bias value for the channel is added to every element in that channel of the output tensor.
```

Comment:
According to the previous [comment](https://github.com/webmachinelearning/webnn/issues/28#issuecomment-539256853) by @BenjaminPoulain, 

> fusing bias and activation should be done by the optimizer.

so we decided to leave bias to `add` operation.

I remember tf.js team has similar opinion, am I correct? @pyu10055

Reply:
Thanks. I'm ok with leaving bias out.

Reply:
Thanks!